lib/src/source_visitor.dart

```diff
@@ -108,12 +108,45 @@ class SourceVisitor extends ThrowingAstVisitor {
 
   visitAdjacentStrings(AdjacentStrings node) {
     builder.startSpan();
-    builder.startRule();
-    visitNodes(node.strings, between: splitOrNewline);
+    var rule = new Rule();
+    builder.startRule(rule);
+
+    var chunk = zeroSplit();
```

Comment:
> How do you intend to handle adjacent strings in `return` and `throw`? I would not expect it to push the first string to the next line there (and we definitely *don't* want it to if we're going to do optional semicolons ever).

That was my intention. It's the only way to keep the strings aligned, unless we try to match human formatting by adding `"return ".length`/`"throw ".length` spaces of indentation.